doc/source/snippets/news-file-format.rst

```diff
@@ -1,34 +1,96 @@
 Why do I need a news file for each PR?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-We want to write good ``CHANGELOG.rst`` for each release version. These news items are of interest to both developers and technical users looking for specific keywords.
+We want to write a good ``CHANGELOG.rst`` for each release version. These news items are of interest to both developers and technical users looking for specific keywords. We can streamline the process of writing ``CHANGELOG.rst`` for each release by compiling the news items from the ``news`` directory.
 
-We can streamline the process of writing ``CHANGELOG.rst`` for each release by compiling the news items from the ``news`` directory.
+Here is an example ``CHANGELOG.rst``: https://github.com/scikit-package/scikit-package/blob/main/CHANGELOG.rst
 
-Here is an example ``CHANGELOG.rst`` https://github.com/scikit-package/scikit-package/blob/main/CHANGELOG.rst
+How do I create a news file?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-How do I write good a news item?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#. In your repository, confirm that the ``news/TEMPLATE.rst`` file exists.
 
-- Do not remove ``news/TEMPLATE.rst``. Make a copy called ``<branch-name>.rst``.
-- Do not modify other section headers in the rst file. Replace ``* <news item>`` only.
-- Start with a capital letter and a verb. End with a period, e.g., ``Add automatic linting of .md, .yml, .rst files via a prettier hook in pre-commit.``
-- For trivial changes, still create ``<branch-name>.rst``, but the news item should start with ``* No news:`` so that the news item is not compiled into the ``CHANGELOG.rst`` file during the release process. Here is an example below:
+#. Make a copy of ``news/TEMPLATE.rst`` and name it ``news/<branch-name>.rst``. You can automate the process of creating the news file and adding new items by running the commands:
+
+    .. code-block:: bash
+
+        $ package add news --add -m "<Describe the first feature added in the PR.>"
+        $ package add news --a -m "<Describe the second feature added in the PR.>"
```

Comment:
it would probably a richer example if you added something and fixed something, for example, rather than added two things

Also, good practice is one thing per branch, so by adding two things is rare when things are being done right.  Don't we want to model that behavior?  I think that the example should just be one thing.  Then in a "note" we can say something like ," it is bad practice in general, but if for some reason two tasks were done on the same branch you can use the shortcuts....", sthg like that.

Reply:
>  it is bad practice in general, but if for some reason two tasks were done on the same branch you can use the shortcuts....", sthg like that.

Yes, one news file created. Also, added the note